integration-tests/utils.py

```diff
@@ -261,6 +261,14 @@ def all_units_ready(model):
                 yaml.dump(data, f)
         await model.deploy(bundle_dir)
     await wait_for_ready(model)
+    # In localhost we need proxy-extra-args="proxy-mode=userspace"
+    if is_localhost():
+        workers = model.applications['kubernetes-worker']
+        new_config={
+            'proxy-extra-args': 'proxy-mode=userspace'
+        }
+        await workers.set_config(new_config)
+        await wait_for_ready(model)
```

Comment:
Can we set this config sooner? Right now it looks to me like we do this:
1. deploy
2. wait_for_ready
3. set_config proxy-mode=userspace
4. wait_for_ready

It seems to me like it would be safer, and save time, to set it immediately after deploy and before the first `wait_for_ready`. Thoughts?

Reply:
Done!